salt/_modules/metalk8s_volumes.py

```diff
@@ -226,7 +226,10 @@ def format(self, force=False):
         if device_info.has_partition:
             raise Exception('backing device `{}` contains a partition table'\
                             .format(self.path))
-        params = self.get('spec.storageClassName.parameters')
+        storage_class = self.get('spec.storageClassName')
+        if isinstance(storage_class, basestring):
```

Comment:
Much confused here, could we have a comment explaining why the negative case is a `basestring`? It's not immediately obvious since we're departing from the usual `get` behaviour.

Reply:
Yeah, deserve a comment indeed.